ui-reference.rst

```diff
@@ -90,27 +90,124 @@ Edit
 (14) **Default Saveconfig:** Typically ignore this. Sets the SaveConfig field on wireguard config.
 (15) **UDP Hole Punching:** Whether or not UDP Hole Punching is turned on (see Network Create notes). Only effects new nodes. Enables or disables "UDP Hole Punching" on the network. When on, clients will first reach out to the server. The server will keep track of public addresses / ports and send these to all other clients in the network. This increases NAT traversibility, but can also cause issues depending on the server environment (if server is in a private network, for example). Typically good to enable if clients will "roam" frequently or are user devices. Typically better to disable if most clients will be servers with well-defined endpoints / ports. If enabled, you can also disable UDP Hole Punching on any individual machine via the UI (see Node section) but it will be enabled by default.
 
+
+Hosts
+======
+
+In simple terms, a host is a netclient or a machine running the netclient software. Netmaker UI allows an admin to conviniently view and configure some host settings remotely.
+
+Host List
+---------
+
+.. image:: images/hosts.png
+   :width: 80%
+   :alt: hosts list
+   :align: center
+
+(1) **Hosts Tab:** Opens a view to allow admins manage hosts.
+(2) **Relay Status:** Indicates whether a host is a relay or is being relayed by a different host on any network.
+(3) **Version:** Indicates the version of netclient the host is running.
+(4) **Default Node:** Click to make the host a default node or not. Hosts that are default nodes will automatically join any created network.
+(5) **Search:** Search for a host by name.
+
+
+Host Create
+-----------
+
+A host is automatically created on a server once a netclient (a machine running netclient) joins any network on the server.
+
+Host Details
+------------
+
+.. image:: images/host-details.png
+   :width: 80%
+   :alt: host details
+   :align: center
+
+The following information is present under the host details tab:
+
+(1) **ID:** Unique identifier for the host
+(2) **Name:** Name of the host. Defaults to the machine's name.
+(3) **Version:** Version of netclient the host is running.
+(4) **Operating System:** Operating system (OS) the machine is running.
+(5) **Public Key:** Public key of the host. distributed to other hosts.
+(6) **MTU:** Maximum Transmission Unit (MTU) of the host
+(7) **Listen Port:** The wiregaurd listen port.
+(8) **Proxy Listen Port:** The netclient proxy listen port. this is used if `Proxy Enabled` is set to `true`.
+(9) **Verbosity:** Log verbosity (ranges from 1-4). Indicates detailed the host (netclient) will output logs.
```

Comment:
Indicates level of detail the host (netclient) will output to logs.

Reply:
rectified